esp32s3-hal/Cargo.toml

```diff
@@ -39,11 +39,13 @@ critical-section  = "1.1.0"
 embedded-graphics = "0.7.1"
 esp-backtrace     = { version = "0.2.0", features = ["esp32s3", "panic-handler", "exception-handler", "print-uart"] }
 esp-println       = { version = "0.3.0", features = ["esp32s3"] }
+sha2              = { version = "0.10.6", default-features = false}
```

Comment:
@jessebraham what do you think about another dev dependency here (because of the CI build times)? I think it's nice to have a comparison of the accelerator's speed vs pure software implementation

Reply:
Yeah in this case I think it's fine, it serves a real purpose and adds something to the example. It doesn't have too many of its own dependencies either.